tests/client/rpc_client.cpp

```diff
@@ -38,7 +35,7 @@ ebpf_rpc_verify_program(ebpf_program_verify_info* info, const char** logs, uint3
     }
     RpcEndExcept
 
-    printf("ebpf_rpc_verify_program: got return code %d from the server\n\n", result);
+        printf("ebpf_rpc_verify_program: got return code %d from the server\n\n", result);
```

Comment:
why is this indented?

Reply:
format code script is causing this. One more thing that I observed is that the format code scripts messes with the IDL file -- it adds spaces between the "-" in the GUID description which causes build error.